prefeRx/src/test/java/com/vinted/preferx/EnumSerializerTest.kt

```diff
@@ -21,6 +21,8 @@ class EnumSerializerTest {
 
     @Test
     fun deserialize_noValue_defaultValue() {
```

Comment:
When mocking always send type default value unless specified in `when`. So, it's always be 0.

Reply:
Could you improve test to know if fix actually works?

Reply:
I think we cannot test this with Mocking unless Roboelectric right?

Reply:
Probably yes. But Robolectric also gives a quality bump. It will test against real or semi-real implementation of shared prefs